### PR TITLE
Add instructions for starting new blog posts with front-matter

### PR DESCRIPTION
**Pull request by kakirastern**
To test whether the `blog: true` flag in the front-matter words for markdown files in creating a new blog post.

### CODE REVIEW

**Comment by nabobalis** [comment]
I can't find the page on the build docs. I am not aware the docs are setup to support markdown.

**Comment by kakirastern** [comment]
Yeah, was just playing around. Don't think markdown is supported here yet...

**Comment by kakirastern** [comment]
I will update the contents of the post once we can see the blog post showing

**Comment by kakirastern** [comment]
@nabobalis I think I will follow up with another PR for the accompanying docs for #69, cause I would like to spend some time to think over it before start writing it, based on the code at https://github.com/sunpy/ablog/blob/master/ablog/blog.py. I will have it done maybe over the weekend.

**Comment by nabobalis** [comment]
Sure but what about this PR?

**Comment by kakirastern** [comment]
Think it depends on whether you would like to keep it as a test.

**Comment by kakirastern** [comment]
I am in favor of keeping it...

**Comment by kakirastern** [comment]
Don't think we have a sample blog post yet

**Comment by nabobalis** [comment]
The content of this blog post is not useful for ablog.

**Comment by kakirastern** [comment]
Yeah, true. Hmm. How about if I modify the contents to make it relevant to #69?

**Comment by nabobalis** [comment]
But is that not at this point writing docs for it?

**Comment by kakirastern** [comment]
Yup, I understand your concerns and they are very valid. I was thinking the docs will be `post`s by construction but not `blog` posts, so this blog will serve as an example to set some pattern. How about I come back to this PR later before end of the coming weekend, to edit it so that it will come with both the docs the explain the context and the blog post with relevant contents? Would that be okay with you?

**Comment by nabobalis** [comment]
Sure.

**Comment by kakirastern** [comment]
Awesome! 😎 I will let you know once this PR is ready for a review then.

**Comment by kakirastern** [comment]
@nabobalis I have now removed the test RST blog post but have added instructions on how to start new blog posts using front-matter rather than directives at `docs/manual/ablog-quick-start.rst`, as I agree with you that this would be more appropriate. So the front-matter that tested to have worked have been moved and used as an example instead.

File: docs/manual/ablog-quick-start.rst [hunk 1]
```diff
@@ -72,13 +72,31 @@ You can convert any page to a post with a :rst:dir:`post` directive.
...
+
+:blogpost: true
+:date: January 1, 2020
+:author: A. Author
+:location: World
+:category: Blog
+:language: English
+:tags: blog
+
+The above assumes that you will be placing the ``.rst`` file for the blog post inside the ``docs/Blog`` directory.
```
Comment (nabobalis) [inline docs/manual/ablog-quick-start.rst:89]:
```suggestion
```

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Looks good.

**Comment by kakirastern** [comment]
Thanks @nabobalis! Appreciate it

**Comment by choldgraf** [comment]
Just jumping in here to say that I was just thinking about how I didn't document that feature addition - thanks much for documenting it for me! ✨

**Comment by kakirastern** [comment]
@choldgraf No problem! It's good to serve as an external check to the entire process as we discovered that additional setup needs to be made on the `sunpy` core config to make the `ablog` front-matter work as it should.

**Comment by nabobalis** [comment]
Did we?

**Comment by kakirastern** [comment]
Right, we haven't fixed the config on the SunPy side just yet 😳

**Comment by nabobalis** [comment]
What was it?

**Comment by kakirastern** [comment]
If we use front-matter for blog post in sunpy.org, currently the posts generated are all automatically labelled as "Draft" posts and are shown as such on our website. Which is not complete since these posts are supposed to be "Blog" posts...

**Comment by nabobalis** [comment]
Drafts? Maybe my browser hasn't refreshed right but I can't see them.

**Comment by kakirastern** [comment]
Oh no worries, the PR has already been merged by you earlier 👍🏼

**Comment by nabobalis** [comment]
So do we have an config issue on sunpy.org or not?

**Comment by kakirastern** [comment]
Not an issue per se, more like a feature

**Comment by nabobalis** [comment]
A bug?

**Comment by kakirastern** [comment]
Yup, think you can say that

**Pull request merged**